### DISCUSSION TITLE
Perfomance - Redis integration, update php8-fpm, better nginx config

### DISCUSSION THREAD

**Discussion by adammakowskidev** [demyxco/demyx#23] [Ideas]
Hi
Demyx you did a very good job!
I like it and I'm testing it constantly

I wonder why there is no redis for cache objects?
I tested php-nginx stack performance on the server 1 cpu 1 gb ram
I got the result of over 900 requests per second
it's not bad but I think you can achieve a lot more
example:
1. add redis object cache
2. update php to php8-fpm
3. config nginx could be much better - https://github.com/denji/nginx-tuning

these are such ideas quickly :)
I am a fan of performance testing and I would be happy to help, I think how to add a brick to the project
Regards

**Comment by demyxco** [comment]
Hi Adam,

Thank you for using Demyx, redis caching isn’t on the roadmap yet since I’m trying to get Demyx as stable as possible. When I release 1.4.0, it will be a performance update. A lot of code will be relooked and refactored.

As for PHP 8, not sure if all plugins support it yet so I’m gonna leave that one out. Although with 1.4.x, there will be an option for you to choose php versions of the docker images.

Lastly, a good way to help is to just use my project and report any bugs/issues you see. Let me know if you have any other questions or feel free to join us in the IRC network of Libera.chat: https://web.libera.chat/?channel=#demyx <https://web.libera.chat/?channel=#demyx>

**Comment by demyxco** [comment]
Hello @adammakowskidev v1.4.0 is here. PHP/LSPHP 8 is now the default. I'll maybe think about adding Redis caching too but will need feedback about the new release first.
